README.md

```diff
@@ -40,6 +40,17 @@ Some of these blocklists are automatically [converted](https://github.com/Adguar
 - If the blocklist intentionally blocks or restricts access to a service for no reason other than being a reflection of the filter author's opinion, the blocklist will not get added, or will get removed if already added.
 - If the blocklist is popular in a specific region and there are no alternatives to it, then it can be added as is even if it does not satisfy requirements above.
 
+## <a id="how-to-add-a-filter-list"></a> How to add a filter list
+
+To add a new filter, you need to:
+
+- Go to the filters folder.
+- Go to the category to which the new filter will belong.
+- Create a new folder with the name "filter_(id)_(filter name)".
+- Create two files in this folder: configuration.json and metadata.json, the contents of these files are described below.
+- In the folder locales/en/filters.json add an object with the name and description of the filter according to the example added above.
```

Comment:
```suggestion
- In the file`locales/en/filters.json` add an object with the name and description of the filter according to the example added above.
```

Reply:
I'm not sure about numerical order, the last sentence is not a step of instruction

Reply:
Just write it without `-`